src/dymoprint/dymo_print_engines.py

```diff
@@ -353,6 +355,9 @@ def merge_render(self, bitmaps, min_payload_len=0, justify="center"):
         else:
             label_bitmap = bitmaps[0]
 
+        if max_payload_len is not None and label_bitmap.width > max_payload_len:
+            die("Error: Label exceeds fixed length")
```

Comment:
```suggestion
            die(
                f"Error: Label length of {label_bitmap.width} pixels exceeds "
                f"allowed length of {max_payload_len}."
            )
```

Reply:
Thought about this, but the command line is in mm and by the time you get here you're in pixels less a margin, so I decided it was on balance less confusing not to give the numbers.